packages/teams-js/src/private/interfaces.ts

```diff
@@ -271,3 +271,142 @@ export interface UserJoinedTeamsInformation {
+ */
+export interface AppEligibilityInformation {
+  /**
+   * Describes the user’s chat experience based on their eligible licenses & their tenant’s eligible licenses.
+   * A user will be in at most one cohort.
+   */
+  cohort: Cohort;
+  /**
+   * Describes additional traits of the user that contribute to FRE experience, etc.
+   */
+  persona: Persona;
+  /**
+   * Describes the user’s age group, which can have implications on which product they are able to use.
+   */
+  ageGroup: LegalAgeGroupClassification;
+  /**
+   * Implementation may change to be based on tenant-home region rather than IP.
+   */
+  isCopilotEnabledRegion: boolean;
+  /**
+   * Indicates that the user is eligible for some AAD authenticated Copilot experience.
+   */
+  isCopilotEligible: boolean;
+  /**
+   * Indicates if the tenant admin has opted the user out of Copilot.
+   */
+  isOptedOutByAdmin: boolean;
+  /**
+   * For EDU tenants only. Indicates if the tenant is higher ed or K12.
+   */
+  eduType: EduType;
+}
+
+/**
+ * @hidden
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * @beta
+ */
+export enum Cohort {
+  None = 'none',
+  BCAIS = 'bcais',
+  BCWAF = 'bcwaf',
+  BCWBF = 'bcwbf',
+}
+
+/**
+ * @hidden
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * @beta
+ */
+export enum Persona {
+  None = 'none',
+  /**
+   * User has a student license
+   */
+  Student = 'student',
+  /**
+   * User has a faculty license
+   */
+  Faculty = 'faculty',
+}
+
+/**
+ * @hidden
+ *
+ * @internal
+ * Limited to Microsoft-internal use
+ *
+ * @beta
+ */
+// https://learn.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-1.0#legalagegroupclassification-values
+export enum LegalAgeGroupClassification {
+  /**
+   * Default value, no ageGroup is set for the user.
+   */
+  None = 'none',
+  /**
+   * Reserved for future use
+   */
+  MinorWithoutParentalConsent = 'minorWithoutParentalConsent',
+  /**
+   * The user is considered a minor based on the age-related regulations of their country or region, and the administrator
+   * of the account obtained appropriate consent from a parent or guardian.
+   */
+  MinorWithParentalConsent = 'minorWithParentalConsent',
+  /**
+   * The user is a minor but is from a country or region that has no age-related regulations.
+   */
+  MinorNoParentalConsentRequired = 'minorNoParentalConsentRequired',
+  /**
+   * The user is considered an adult based on the age-related regulations of their country or region.
+   */
+  Adult = 'adult',
+  /**
+   * The user is from a country or region that has additional age-related regulations, such as the United States,
+   * United Kingdom, European Union, or South Korea, and the user's age is between a minor and an adult age
+   * (as stipulated based on country or region). Generally, this means that teenagers are considered as notAdult in regulated countries.
+   */
+  NonAdult = 'nonAdult',
+  /**
+   * Value is not found.
+   */
+  Unknown = 'unknown',
```

Comment:
Are consumers of this code supposed to treat the "unknown" case different than the "None" case?